samba/run.sh

```diff
@@ -5,6 +5,14 @@ INTERFACE=$(bashio::config 'interface')
 ALLOW_HOSTS=$(bashio::config "allow_hosts | join(\" \")")
 USERNAME=$(bashio::config 'username')
 PASSWORD=$(bashio::config 'password')
+VETO_FILES=$(bashio::config "veto_files | join(\"/\")")
```

Comment:
This could be written as:

```bash
VETO_FILES=$(bashio::config "veto_files | join(\"/\")")
DELETE_VETO_FILES="no"
if bashio::config.has_value 'veto_files'; then
    VETO_FILES="/$VETO_FILES/"
    DELETE_VETO_FILES="yes"
fi
```